examples/exporting_for_inference.ipynb

```diff
@@ -75,89 +59,243 @@
   },
   {
    "cell_type": "markdown",
-   "id": "8f4bd162",
+   "id": "87cc6bc6",
    "metadata": {},
    "source": [
-    "Printing the model shows the new `Bottleneck` layers:"
+    "## Torchscript Export using standalone API\n",
+    "Torchscript creates models from PyTorch code that can be saved and also optimized for deployment, and is the tooling is native to pytorch. \n",
+    "\n",
+    "The *ComposerClassifier*’s forward method takes as input a pair of tensors (input, label), so we create a dummy tensors to run the model."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "8b876784",
+   "id": "02603737",
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(model)"
+    "import torch\n",
+    "\n",
+    "input = (torch.rand(4, 3, 224, 224), torch.Tensor())\n",
+    "\n",
+    "output = model(input)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "8517a2d2",
+   "id": "5668ffed",
    "metadata": {},
    "source": [
-    "## ONNX\n",
+    "Now we run export using our standalone export API. Composer also supports exporting to an object store such as S3. Please checkout [full documentation](https://docs.mosaicml.com/en/stable/api_reference/composer.utils.inference.html) for `export_for_inference` API for help on using an object store."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e654bff1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import tempfile\n",
+    "from composer.utils import export_for_inference\n",
     "\n",
-    "[ONNX](https://onnx.ai/) is a popular model format that can then be consumed by many third-party tools (e.g. TensorRT, OpenVINO) to optimize the model for specific hardware devices. \n",
+    "save_format = 'torchscript'\n",
+    "working_dir = tempfile.TemporaryDirectory()\n",
+    "model_save_path = os.path.join(working_dir.name, 'model.pt')\n",
     "\n",
-    "Note: ONNX does not have a prebuild wheel for Mac M1/M2 chips yet, so is not pip installable. Skip this section if you are running on a Mac laptop."
+    "export_for_inference(model=model, \n",
+    "                     save_format=save_format, \n",
+    "                     save_path=model_save_path, \n",
```

Comment:
New line and comma probably not needed?
You can close the parenthesis on same line I guess.

Reply:
fixed.